lms/djangoapps/instructor/enrollment.py

```diff
@@ -417,7 +417,10 @@ def render_message_to_string(subject_template, message_template, param_dict, lan
     Returns two strings that correspond to the rendered, translated email
     subject and message.
     """
-    with override_language(language):
+    if language is not None:
```

Comment:
You should add some comments here explaining what you're doing and why. Perhaps here it would be,
```python
# Override the email language with the user's preference, if it is set
if language is not None:
```